Sources/MarathonCore/ScriptManager.swift

```diff
@@ -94,7 +94,7 @@ internal final class ScriptManager {
 
     // MARK: - API
 
-    func script(at path: String) throws -> Script {
+    func script(at path: String) throws -> Script {  // TODO: change param to `atPath`
```

Comment:
`script(at:)` should be renamed to `script(atPath:)` since the argument type is String which gives no context.